src/constants/contactForm.ts

```diff
@@ -13,7 +13,7 @@ export const SUBMIT_BUTTON_IMG_PATH = "/imgs/circled-arrow-icon.svg";
 export const MAX_NAME_LENGTH = 100;
 export const MAX_EMAIL_LENGTH = 100;
 export const MAX_MESSAGE_LENGTH = 6000;
-export const EMAIL_PATTERN = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i;
+export const EMAIL_PATTERN = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
```

Comment:
Do you even need the following '{2,}' any longer? Not sure what you are matching with this? :)

Reply:
I see what it is doing. It is matching 2 of anything, is that correct?

Reply:
Domain extensions are 2 char or longer, 1 char is invalid, I think it's fine to have it like this.

Reply:
> I see what it is doing. It is matching 2 of anything, is that correct?

Yes, 2 or more of any char for the domain extension

Reply:
Perfect. I did some tests, it works great!